app/controllers/dois_controller.rb

```diff
@@ -689,14 +691,23 @@ def safe_params
     read_attrs_keys.each do |attr|
       p.merge!(attr.to_s.underscore => p[attr] || meta[attr.to_s.underscore] || p[attr]) if p.has_key?(attr) || meta.has_key?(attr.to_s.underscore)
     end
+
+    # handle alternate_identifiers
+    p[:identifiers] = Array.wrap(p[:alternate_identifiers]).map do |a|
```

Comment:
I think this will replace `identifier` with `alternateIdentifier`. Shouldn't we cope with both cases?

Reply:
What is the other case? Users submitting `identifiers` to the REST API?

Reply:
exactly. This was previously mentioned in support https://app.frontapp.com/open/msg_bban6tf

Reply:
To be clear, the support message is not about submitting `identifiers`. We can clarify that `alternateIdentifiers` should be used.